packages/client/components/PalettePicker/PalettePicker.tsx

```diff
@@ -0,0 +1,90 @@
+import React, {useEffect, useState} from 'react'
+import {TemplatePromptItem_prompt} from '../../__generated__/TemplatePromptItem_prompt.graphql'
+import PaletteColor from '../PaletteColor/PaletteColor'
+import Menu from '../Menu'
+import {MenuProps} from '../../hooks/useMenu'
+import withAtmosphere, {WithAtmosphereProps} from '../../decorators/withAtmosphere/withAtmosphere'
+import withMutationProps, {WithMutationProps} from '../../utils/relay/withMutationProps'
+import ReflectTemplatePromptUpdateGroupColorMutation from '../../mutations/ReflectTemplatePromptUpdateGroupColorMutation'
+import styled from '@emotion/styled'
+import {PALETTE} from '../../styles/paletteV2'
+import {palettePickerOptions} from '../../styles/palettePickerOptions'
+
+interface Props extends WithAtmosphereProps, WithMutationProps {
+  prompt: TemplatePromptItem_prompt
+  prompts: any
+  menuProps: MenuProps
+}
+
+const PaletteDropDown = styled(Menu)({
+  width: '214px',
+  minWidth: '214px',
+  padding: 5
+})
+
+const PaletteList = styled('ul')({
+  display: 'flex',
+  flexWrap: 'wrap',
+  justifyContent: 'center',
+  listStyle: 'none',
+  padding: 0,
+  margin: 0
+})
+
+const PaletteItem = styled('div')({
+  cursor: 'grab'
```

Comment:
I felt it looked cleaner this way really. Since the cursor is set to 'grab' when the palette opens, and rather than have the cursor jump between 3 different styles (pointer, grab, default) when traversing the palette, I felt that the pointer action would help to indicate the desired click. However, since this is on top of an actual grabbable/draggable component, I can see that this could potentially mislead the user of the intended action. I changed it to use default now in between hover states.